modules/team-member.php

```diff
@@ -1,22 +1,31 @@
 <?php
 	$profile_picture = get_field('profile_picture');
+	$prof_pic = get_field('prof_image');
```

Comment:
Could the variable names `$prof_pic`, `$profile_picture`, and `$profile_image` be more descriptive? For example, something like `$profile_pic_media`, `$profile_pic_url`, and `$parsed_profile_pic_url` respectively?

Same goes for video-thumbnail.php.

Reply:
For sure! @FaraiMajor, please feel free to change the variable names

Reply:
Thanks @smrohrer I will get right to it